packages/saved-views-client/src/models/savedViews/DisplayStyles.ts

```diff
@@ -702,7 +702,7 @@ export interface PlanarClipMaskProps {
   mode: PlanarClipMaskMode;
   modelIds?: string;
   subCategoryOrElementIds?: string;
-  priority?: PlanarClipMaskPriority;
+  priority?: number;
```

Comment:
Add changelog entry to saved-views-client CHANGELOG.md

Reply:
Added changelog for this. :)